llvm/lib/Analysis/IR2Vec.cpp

```diff
@@ -280,6 +291,7 @@ StringRef Vocabulary::getVocabKeyForTypeID(Type::TypeID TypeID) {
   default:
     return "UnknownTy";
   }
+  return "UnknownTy";
```

Comment:
This return seems redundant given there's a default path in the switch?

Reply:
Compiler will throw warnings without return